vncdotool/command.py

```diff
@@ -172,7 +172,7 @@ def build_command_list(factory, args, delay=None, warp=1.0):
                         imgformat, SUPPORTED_FORMATS), file=sys.stderr)
                 sys.exit(1)
             else:
-                factory.deferred.addCallback(client.captureScreen, filename)
+                factory.deferred.addCallback(client.captureScreen, filename, int(incremental_refreshes))
```

Comment:
The type conversion here feels a bit strange.  Can we avoid this somehow, do it earlier, later or trust duck typing to do the right thing?

Reply:
The function expecting an int value and the flag is boolean, do you think that we need to convert it before?